### ISSUE TITLE
Clear definition of requirements for standards in the DPG standard

### ISSUE THREAD

**Issue by christer-io** [DPGAlliance/dpg-standard#126]
The DPG standard defines standards as one of the categories of possible nominations to become PDGs. 

The 9 indicators cover content, open source and data very well, but has no reference to actual criteria's that would give a clear frame for standards. 

I suggest a quick iteration to fix this with focus on:
- What defines an open standard
- Other requirements like reference implementation, open governance, transparent processes and vendor/platform independence

**Comment by christer-io** [comment]
Based on our conversation in the standard council last week I have carved out the following suggest next steps:
1. Remove standard as a one of the main categories of DPGs that can be nominated 
2. Rewrite indicator 8 with language to resemble indicator 2:	
- Language like….“Digital public goods must demonstrate the use of open standards when relevant”.
- And….”The DPGA refer to the following standardization organizations maintaining relevant standards” 
- More clear direct link to the 5 Core Principles of OpenStand from the standard text (we refer to them on our Github repo today but not in the standard text)

https://open-stand.org/about-us/principles/

## Organizations that we could refer to:

### World Wide Web Consortium
The World Wide Web Consortium (W3C) is an international community that develops open standards to ensure the long-term growth of the Web. 

https://www.w3.org/

### International Organization for Standardization (ISO)
ISO is an independent, non-governmental international organization with a membership of 167 national standards bodies.

https://www.iso.org/home.html

### ITU Telecommunication Standardization Sector (ITU-T)
The ITU Telecommunication Standardization Sector (ITU-T) coordinates standards for telecommunications and Information Communication Technology such as X.509 for cybersecurity, Y.3172 and Y.3173 for machine learning, and H.264/MPEG-4 AVC for video compression, between its Member States, Private Sector Members, and Academia Members.

https://www.itu.int/

### Internet Engineering Task Force
The Internet Engineering Task Force (IETF) is an open standards organization, which develops and promotes voluntary Internet standards, in particular the technical standards that comprise the Internet protocol suite (TCP/IP). It has no formal membership roster or membership requirements. All participants and managers are volunteers, though their work is usually funded by their employers or sponsors.

https://www.ietf.org/

**Comment by prajectory** [comment]
Why is Open Standard suspended as a DPG category? 

Context: 
Today, solutions reviewed through the DPG Standard include open-source software, open data, open AI models, open standards, and open content of relevance to attainment of the Sustainable Development Goals.

After running the review process for two years, we have identified several challenges with including open standards as a distinct category of digital public goods as compared to the other categories.  Below, some key considerations are provided that have led to the proposal currently being presented to the Board.

**Comment by prajectory** [comment]
Why is it a challenge to assess open standards as a DPG category? 

Properly assessing a proposed open standard from a perspective of ensuring it does not ultimately undermine the openness of a sector or industry, is extremely challenging, and would at a minimum require deep knowledge of the standard’s governance and the sectoral context in which the standard will operate. 

Otherwise some of the below mentioned features of digital public goods do not get fulfilled. 

Adoptability and adaptability: A key property of digital public goods, as per how the term originated in the report from the High-level Panel on Digital Cooperation as well as the definition in the UN Secretary-General’s High-level Panel on Digital Cooperation, is their adoptability and adaptability. 

No Vendor lock-in: Furthermore, a key consideration of the DPGA in the advancement of digital public goods is to ensure countries avoid long-term vendor lock-ins as part of their digital transformation efforts. While standards are necessary for ensuring better interoperability and a more open digital ecosystem, even open standards can be implemented in ways that result in long-term vendor lock-in, including in combination with proprietary software.

The DPG review team is not equipped to ensure the above two features are integrated well into "open standard" as a DPG be cause, the standard itself is not optimised for reviewing standards as a distinct category, for example the DPG standard does not mandate documentation to be submitted for how the governance of a standard works - a critical topic for assessing ‘open standards’. 

Furthermore, from a digital technology adoption perspective, standards will typically be implemented by stakeholders in combination with one or several of the other DPG categories (software, content, data, AI-models) as part of a use case, so it also gives more meaning from a user perspective to focus on standards as integral to other DPG categories than as a stand-alone category.

**Comment by prajectory** [comment]
Solution? 
In order to assess the value of a digital solution as an “open standard” we would need to develop a set of new criteria that are not necessarily applicable to the other DPG categories, and we would need to build out a vastly more comprehensive Secretariat than what we have today in order to properly conduct assessments against such criteria. 

The two considerations are duplication of efforts and expertise. 

There are many excellent bodies already reviewing and maintaining up-to-date lists of open standards, and we would risk duplicating these efforts.

**Comment by prajectory** [comment]
For this reason, the DPG Standard council proposes that we no longer review open standards as a "category of DPG". The proposal was shared with the DPGA board so the changes could be executed and in parallel a 'standard working group' was setup to delve deeper into the subject. 

If the proposal is accepted by the Board, open standards will remain a critical component of DPGs but not be categorised as DPGs in and of themselves. Open standards will be more strongly emphasised in our list of requirements/best practices for new submissions for other categories of DPGs, and we will increase discoverability of recognized open standards best practices and defer to established standards bodies where possible.

**Comment by prajectory** [comment]
The DPGA board reviewed the proposal to permanently suspend 'open standard' as a category. The majority voting led to most board members approving the proposal while some expressed the need to dive deeper to understand "open standard as a DPG" better. 

The Standard Council constituted a group of experts into a "open standard working group". 

The working group assessed various examples of open standards against the text and spirit of the DPG standard to understand in-depth how each indicator relates to open standard and whether it fulfils the criteria as a digital public good. In order to be inclusive the working group looked at both technical and non-technical standards (specifications).

**Comment by prajectory** [comment]
Final conclusion from the open standard working group and the standard council and opening up for **Community Discussion Period** of 2 weeks: 

After running the review process for two years and going over 5+ applications that applied as an open standard, we have identified several challenges with including open standards as a distinct category of digital public goods as compared to the other categories. The reasons for the same are enumerated in observations above. 

We are suspending open standard as a category of digital public good. 

The DPG Standard Council instead recommended a larger expert consultation to include ‘open standard’ in a more fundamental way and consider adding it as a documentation requirement for some or all DPGs. 

While we deliberate this further, **we invite the community to add their thoughts on**:

1. Suspension of open standards as a DPG type
2. Delving deeper into how “open standards” can become a more fundamental part of the DPG standard by becoming referencing or documentation requirement

**Comment by grahamegrieve** [comment]
I agree with the criteria from https://open-stand.org/about-us/principles/, but I'm surprised that they aren't explicit that there cannot be IP encumbrances to implementing the standard, or making code that implements the standard open source. (I think they should say that in addition to "Given market diversity, fair terms may vary from royalty-free to fair, reasonable, and non-discriminatory terms (FRAND).")

Also, they talk about voluntary adoption. That's tricky - global standards might be voluntary in many contexts, but mandated in some. It's pretty tricky to nail that one down. 

In principle, I'm OK with not making open standards a DPG type, but it seems appropriate to have a process for registering a standard as appropriate for adopting in a DPG - .e.g a separate category

**Comment by jpmckinney** [comment]
Noting that a "Standard" checkbox appears under the "Type" filter (though it matches no items): https://digitalpublicgoods.net/registry/

----

> Suspension of open standards as a DPG type

As I understand, the main reason for the removal of "open standards" is practical, rather than conceptual: the DPG review team is not equipped to review standards. Fair enough!

> Delving deeper into how “open standards” can become a more fundamental part of the DPG standard by becoming referencing or documentation requirement

I think a new issue should be opened for this. This issue started on the subject of requirements, and ended with the suspension of open standards. Adding a third subject makes it very confusing to new contributors.

---

The rejection message for submitting a standard contains:

> If there are any open datasets that you own related to this standard please create a new submission for those.

I'm not sure this suggestion makes sense. Open standards tend to be voluntarily adopted by independent organizations. A standards author might aggregate datasets published according to their standards, but they would not "own" these. A standards author could contact their adopters to submit directly, but that would yield thousands of submissions... I'm not sure who the audience is for this sentence.

---

> making code that implements the standard open source

What do you mean? Something like copyleft, where any implementer of the standard must make their code open source?

**Comment by jpmckinney** [comment]
Also, on the submission form, it might be a good idea to explicitly state "We are currently not accepting open standard submissions" near the dropdown for "Solution category".

**Comment by greggish** [comment]
As someone who submitted an open standard for consideration and received a notice to this effect, I understand the reasoning here on a pragmatic basis. That said, standards are quite clearly public goods (especially when provisioned in alignment with the conditions that you understandably have decided you're not equipped to vet). 

And the boundary between standards and the other categories here – data, content, software – can be pretty blurry:
- Our standard enables the production of a specific kind of data as a public good; 
- Our open standard initiative produces open source content to aid in the production of that data as a public good
- Open source software that uses our standards can be used by anyone to produce open data as a public good. 

My attempt to read through this thread carefully leaves me with the impression that you're not trying to say that standards are not digital public goods, just that you're not in a position to evaluate them. Which is fine, but I hope that your communications can still acknowledge this and assert the value of standards as public goods; and that your strategic roadmap can still prioritize standards as a future objective. 

My concern is that ceding this territory indefinitely will leave this initiative with a relatively simplistic scope around solutionist apps that – without entanglement with broader ecosystems of standards and infrastructure – will fall inevitably short of the promise of your vision.
